recOrder/plugin/main_widget.py

```diff
@@ -990,6 +1003,8 @@ def connect_to_mm(self):
             self.calib_mode = "DAC"
             self.ui.cb_calib_mode.setCurrentIndex(2)
 
+        print("Finished connecting")
```

Comment:
The `recOrder` plugin widget provides a visual hint of whether the connecting was successful. We may not need this line.